aqua.yaml

```diff
@@ -7,4 +7,4 @@ registries:
 packages:
 - name: tinygo-org/tinygo@v0.36.0
 - name: WebAssembly/binaryen@version_112
-- name: magefile/mage@v1.14.0
+- name: magefile/mage@v1.14.0
```

Comment:
Do we need to install `mage` in aqua?

Reply:
It's explained in this commit message.
https://github.com/aquasecurity/trivy/pull/8493/commits/4bca3f12a67ceee3f9b671f5182ff553fece1283

Reply:
Since I want to remove the TinyGo dependency and migrate to BigGo, I didn't try to figure out the problem.